spec/avrolution/tasks_spec.rb

```diff
@@ -28,6 +28,24 @@
     end
   end
 
+  describe "register_all_schemas" do
+    let(:task_name) { 'avro:register_all_schemas' }
+    let(:schema_files) { Avrolution::DiscoverSchemas.discover(Dir.pwd) }
+    let(:register_schemas) { Avrolution::RegisterSchemas.new(schema_files) }
```

Comment:
Bit of a nit, but using a real object here is a form of "partial double", when we're only caring about one call. We can simplify slightly by using `instance_spy(Avrolution::RegisterSchemas)` here, to clarify that we don't expect any real calls to the underlying object. This also removes the need for line 39.